http/server.ts

```diff
@@ -683,7 +696,20 @@ export async function serveTls(
     once: true,
   });
 
-  const s = server.listenAndServeTls(options.certFile, options.keyFile);
+  const key = options.key || Deno.readTextFileSync(options.keyFile!);
+  const cert = options.cert || Deno.readTextFileSync(options.certFile!);
```

Comment:
(Side Note: `Deno.readTextFileSync` makes sense to me here because the current version of `serveTls` starts the listener synchronously. This keeps that behavior.)